mutt_config.c

```diff
@@ -817,6 +787,7 @@ static void init_variables(struct ConfigSet *cs)
   CONFIG_INIT_VARS(cs, helpbar);
   CONFIG_INIT_VARS(cs, history);
   CONFIG_INIT_VARS(cs, index);
+  CONFIG_INIT_VARS(cs, pager);
```

Comment:
For all the OCD sufferers out there -- this list is alphabetically sorted :-)

Reply:
fixed. But how do OCD people feel that line 777 is `config_init_main(cs);` and not `CONFIG_INIT_VARS(cs, main);`
That breaks the pattern! :open_mouth: :smile: